packages/api/methods.ts

```diff
@@ -52,7 +52,7 @@ export async function batchBudgetUpdates(func) {
   }
 }
 
-export function runQuery(query) {
+export function aqlQuery(query) {
```

Comment:
This is a breaking change, right? Should we maybe keep the name the original name in the API? Or at least support both for a version or two and call it out in the release notes.

Reply:
Good idea. I have added back the runQuery and added a deprecation note on it instead. We'll just remove it on a future release.